web/modules/custom/joinup_search/joinup_search.module

```diff
@@ -395,3 +396,25 @@ function joinup_search_views_pre_execute(ViewExecutable $view) {
   // Selecting the entity_created option itself won't affect the query.
   $view->getQuery()->sort('entity_created', 'DESC');
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function joinup_search_form_views_exposed_form_alter(array &$form, FormStateInterface $form_state, string $form_id) {
+  $view = $form_state->getStorage('view');
```

Comment:
This method doesn't take an argument, it can just be `->getStorage()`. I saw this during review because PHPStorm highlights this when you enable the inspection "Parameters number mismatch declaration".